### DISCUSSION TITLE
It will be great if it can auto fetch m3u lists from iptv github project

### DISCUSSION THREAD

**Discussion by Greatz08** [4gray/iptvnator#489] [Ideas]
It will make things much easier for beginners in my opinion , As they wont have to go to https://github.com/iptv-org/iptv   to get what they want . So better ui for this feature could help them to get things easily  
Rest your wish  :-))

**Comment by 4gray** [comment]
Yes, I understand your point. Unfortunately, this isn’t possible because of legal considerations around content distribution. IPTVnator is just a player.
